features/polyExplorer/src/screens/stories/story-utils.js

```diff
@@ -74,7 +74,7 @@ export function createDataTypesSharedCombined(entities) {
                 currentCategory.value = currentCategory.value + category.count;
             else {
                 categoriesCount[category.translation] = {
-                    category: category.translation,
+                    title: category.translation,
```

Comment:
The chart should stay neutral to use cases, meaning category is to adapted to one thing rather than a general attribute in my opinion. Nodes have a title/id in every other chart, would be weird to do it differently here. The chart needs to have a key here